support-frontend/assets/pages/aus-moment-map/components/testimonialsContainer.tsx

```diff
@@ -43,18 +48,24 @@ function TestimonialCtaPrimary() {
 function TestimonialCtaSecondary() {
 	return (
 		<div className="testimonial-cta testimonial-cta-secondary">
-			<h3>Do something powerful today</h3>
-
-			<LinkButton
-				className="testimonial-cta-secondary-link-button"
-				priority="primary"
-				size="small"
-				href={contributeUrl('Aus_Moment_2021_map_inline')}
-				target="_blank"
-				rel="noopener noreferrer"
-			>
-				Support the Guardian
-			</LinkButton>
+			<h3>
+				<span className="testimonial-cta-header">
+					Do something powerful today
+				</span>
+			</h3>
+
+			<ThemeProvider theme={buttonThemeBrandAlt}>
+				<LinkButton
+					className="testimonial-cta-primary-link-button"
+					priority="primary"
+					size="small"
+					href={contributeUrl('Aus_Moment_2021_map_inline')}
```

Comment:
Do we still want this to be `2021`?

Reply:
Yeah well spotted 👍 We've got a second ticket for updating all of the tracking links so was going to make that update then

Reply:
Cool cool